graphql-codegen-haskell/src/index.ts

```diff
@@ -71,6 +72,9 @@ export const generate = async (configPath: string): Promise<OutputFiles> => {
     filesToGenerate[modulePath] = content
   }
 
+  // globally disable html-escaping
+  Mustache.escape = x => x
```

Comment:
Can we do this in `renderAPIModule` instead? Would like to keep all mustache-specific logic in one place